docs/voltohandson/breadcrumbs.md

```diff
@@ -15,7 +15,7 @@ myst:
 
 We want to hide breadcrumbs from the homepage and change the styling a bit.
 
-We can do it by using bare styling, since Volto injects CSS classes in the body that help us to style depending on the object, the content type and the path.
+We can do it by using bare styling, since Volto injects CSS classes in the body that help us to style depending on the object, the Content Type and the path.
```

Comment:
We actually should not title case this thing. It is not a proper noun, just an ordinary noun. However some commas will make it easier to read.

```suggestion
We can do it by using bare styling, since Volto injects CSS classes in the body that help us to style, depending on the object, the content type, and the path.
```